celery_progress/websockets/tasks.py

```diff
@@ -12,3 +12,15 @@ def task_postrun_handler(task_id, **kwargs):
     result = KnownResult(task_id, kwargs.pop('retval'), kwargs.pop('state'))
     data = Progress(result).get_info()
     WebSocketProgressRecorder.push_update(task_id, data=data, final=True)
+
+
+@task_revoked.connect(retry=True)
+def task_revoked_handler(request, **kwargs):
+    """Runs if a task has been revoked. This will be used to push a websocket update for revoked events.
+
+    If the websockets version of this package is not installed, this will fail silently."""
+    _result = ('terminated' if kwargs.pop('terminated') else None) or ('expired' if kwargs.pop('expired') else None) \
```

Comment:
Is this the value used in line 79 in get_info? How is it populated for HTTP? I just want to ensure WS and HTTP behave the same because this looks hand-crafted but the HTTP code uses a single source (result.info).

Reply:
This is the `result.info` value that gets used at that line, yes. HTTP populates this value by requesting the backend, which has the same values. The handler for revoked tasks does not pass the result, so I had to manually re-create the intended result in order to maintain parity with HTTP. All of the same data is there, it's just that rather than making a trip to the backend, I've opted to use what I've already been provided to do largely the same thing.

Reply:
Makes sense. Unfortunate that result.info is not well documented in celery.